examples/example_fermi_catalogs.py

```diff
@@ -0,0 +1,97 @@
+""" Example to show how to plot spectrum of Fermi/LAT sources
+"""
+
+"""Example how to plot Fermi-LAT catalog spectra.
```

Comment:
This example file has two docstrings as the top.
I like your enthusiasm for documentation!